source/develop/release-management/features/network/networking-api-security-groups.html.md

```diff
@@ -0,0 +1,352 @@
+
+The security group has the following attributes:
+
+| name                 | type   | description |
+|----------------------|--------|-------------|
+| tenant_id       | string | The ID of the tenant.                                                      |
+| project_id      | string | The ID of the project.                                                     |
+| description (*) | string | A human-readable description for the resource. Default is an empty string. |
+| name            | string | Human-readable name of the resource.                                       |
+
+(*) - optional attributes
+
+All of the aforementioned attributes will be stored in the external_ids
+of the corresponding port group.
+
+### Encoding the security group rule information
+
+The security group information will be encoded in the ACL object, to
+which it naturally corresponds in the OVN-world.
+
+A security group rule has the following attributes:
+
+| name                 | type   | description |
+|----------------------|--------|-------------|
+| remote_group_id (**) | string | The remote group UUID to associate with this security group rule. You can specify either the  remote_group_id or remote_ip_prefix attribute in the request body. |
+| direction            | string | Ingress or egress, which is the direction in which the security group rule is applied.                                                                           |
+| protocol (*)         | string | Check https://developer.openstack.org/api-ref/network/v2/#create-security-group-rule                                                                             |
+| ethertype (*)        | string | Check https://developer.openstack.org/api-ref/network/v2/#create-security-group-rule                                                                             |
+| port_range_max (*)   | int    | The maximum port number in the range that is matched by the security group rule.                                                                                 |
+| port_range_min (*)   | int    | The minimum port number in the range that is matched by the security group rule.                                                                                 |
+| security_group_id    | string | The security group ID to associate with this security group rule.                                                                                                |
+| remote_ip_prefix (*) | string | The remote IP prefix that is matched by this security group rule.                                                                                                |
+| description (*)      | string | A human-readable description for the resource. Default is an empty string.                                                                                       |
+
+(*) - optional attributes
+(**) - optional attributes that ***will not*** be implemented.
+
+### OVN ACL table
+
+An OVN ACL has the following attributes:
```

Comment:
I think this table is useful, since it is hard to refer the user to the ovn-nb documentation - it is a txt, where - afaik - I can't point to a specific anchor in the file